packages/react/src/auth/AuthenticationForm.module.css

```diff
@@ -1,3 +1,7 @@
 .userNotice {
   background-color: light-dark(var(--mantine-color-gray-0), var(--mantine-color-gray-8))
 }
```

Comment:
I think you removed the only usage of this rule, so it can get removed too. I didn't realize you were going to be updating this page when I added this, your version is much more comprehensive than the quick patch I dropped in here to reveal the email address! Awesome :tada: